### PR TITLE
Supports --globals config

### LINKED ISSUE #184: UMD dependencies
When you're working on UMD build, if you're using an external module, the name microbundle uses to import this module can be different from the one the module truly uses.

For instance, React exports window.React but microbundle wants to use window.react.

Can we change it?

### PR DESCRIPTION
**Pull request by Ayc0**
Add --globals config in the cli that allows to rename globals dependencies.

Fixes #184

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,7 +106,6 @@ Here's what's coming up for Microbundle:
 
 [MIT](https://oss.ninja/mit/developit/)
 
-
-[Rollup]: https://github.com/rollup/rollup
-[Bublé]: https://github.com/Rich-Harris/buble
-[Nodent]: https://github.com/MatAtBread/nodent-compiler
```
Comment (ForsakenHarmony) [inline README.md]: why, if anything you should correct `Nodent` to `NoDent`

Comment (Ayc0) [reply README.md]: I didn't change this line, I only added the line for globals but the precommit hook includes lint-staged which modified the rest of the README

Comment (Ayc0) [reply README.md]: Yep, at the beginning, I thought it was due to the prettier extension on my VSCode, and so I've desactivated the format.onSave setting on VSCode

But it behaved the same way when I try to commit after modifying the file with nano...

If you want I can recommit it with --no-verify or without lint-staged

Comment (ForsakenHarmony) [reply README.md]: just seems really weird that it would lower case that and not the link above

Comment (Ayc0) [reply README.md]: 🤷‍♂️

Comment (developit) [reply README.md]: documentation.js does some strange markdown injection stuff. as long as it still works it's probably safe to ignore haha

**Comment by Ayc0** [comment]
I implemented the API this way:
`microbundle --globals npmLibrayName=globalName,other=otherGlobal`

**Comment by cyrilchapon** [comment]
OMFG one has to document this. We've been searching for the way to use lodash UMD for 1 hour.

**Comment by Ayc0** [comment]
@cyrilchapon it's written in the README.md `    --globals        Specify globals dependencies, or 'none'
` and in the `--help` command

**Comment by cyrilchapon** [comment]
Yeah, we just missed the "examples" part.

**Pull request merged**